package.json

```diff
@@ -75,6 +76,7 @@
     "axios": "0.27.2",
     "classnames": "2.3.1",
     "date-fns": "^2.28.0",
+    "file-saver": "^2.0.5",
```

Comment:
we added only one library, but we have 50k changes in lock file?

Reply:
I'll investigate it. I had a problem with installation of the the packages. I'll reset to the package-lock.json from main and try again with the installation.